$NetBSD: patch-requirements.txt,v 1.3 2023/04/01 13:46:16 riastradh Exp $

Relax pycountry dependency until someone can update it in pkgsrc and
make it pass all the tests.

--- requirements.txt.orig	2023-04-01 13:33:41.040763821 +0000
+++ requirements.txt
@@ -6,7 +6,7 @@ intervaltree>=3.1.0
 jinja2>=3.1.2
 markupsafe>=2.1.1
 lxml>=4.9.0
-pycountry>=22.3.5
+pycountry>=19.8.18
 pyyaml>=5.3.1
 requests>=2.5.0
 setuptools>=24.2.0
